### ISSUE TITLE
Logros y reputaciones de hermandad

### ISSUE THREAD

**Issue by paellaencol** [RoDNet/Bugtracker-Infinium-4.3.4#1395]
Buenas noches!

-Los logros de hermandad http://es.wowhead.com/achievements=3 no están funcionando, no me atrevo a decir que no va ninguno, pero hasta el momento ni se activan, ni muestran una progresión

-Matar un boss yendo con miembros de tu hermandad tampoco está dando reputación (en la 4.0.6 funcionaba)

Cito:

Los miembros de la hermandad pueden aportar experiencia al nivel de su hermandad completando misiones, misiones diarias, matando jefes de mazmorras y bandas con grupos compuestos en un 80 % por miembros de la hermandad y ganando encuentros en campos de batalla puntuados con grupos compuesto en un 80 % por miembros de la hermandad.

Hay una excepción a la norma del 80% que son las mazmorras de 5 jugadores, basta con que 3 de los 5 miembros pertenezcan a la misma hermandad para ganar experiencia y logros de hermandad.\* La cantidad de experiencia obtenida va aumentando: si 3 de los 5 son miembros de la hermandad obtendrán el 50% de experiencia de hermandad normal, 4 de 5 obtendrán el 100% y 5 de 5 obtendrán el 125%. La cantidad obtenida por cada jefe va aumentando con tu nivel según el nivel del jefe, con jefes heroicos vale incluso más (se multiplica por 1,5). ¡Merece la pena tener un grupo completo de miembros de la hermandad!

Otra excepción son las antiguas bandas de 40 personas que solo requieren un 25% de participación de la hermandad (10 miembros) para ganar experiencia y logros.

Un saludo

**Comment by Yoshyy** [comment]
Cierto, solo van los logros de hermandad cuando aumenta de nivel a 5, 10 y sucesivamente, pero todos los relacionados con mazmorras etc... con miembros de guild, están bug, espero que puedan arreglarlos pronto, gracias!! ^^
